LINGUAS

```diff
@@ -1,27 +1,26 @@
+ca
 cs
 de
+eo
```

Comment:
Is it possible to not make any changes to this file other than adding your language?

Reply:
I'm just sorting language ids alphabetically.

Reply:
A few languages are missing though, immediately noticeable is the missing "ko" and "oc" for example.

Reply:
You are right, I fixed it in my master branch